### ISSUE TITLE
pip install doesn't work

### ISSUE THREAD

**Issue by Gadgeteering** [rienafairefr/python-ldraw#1]

**Comment by rienafairefr** [comment]
Was this a `ImportError: No module named 'setuptools_scm'` ? 
I pushed a bugfix that should be in 0.8.1, fixing that, sorry.

**Comment by Gadgeteering** [comment]
Yes I downloaded the repository from github and overwrote the files and now it works
